js/Cardz.jsx

```diff
@@ -0,0 +1,102 @@
+import React, { Component } from "react";
+import {connect} from 'react-redux';
+
+import * as actions from '../redux/actions/workspace.js';
+import store from '../redux/store.js';
+import {Card, CardActions, CardHeader, CardMedia, CardTitle, CardText} from 'material-ui/Card';
+import update from 'react-addons-update';
+
+const styles = {
+    cardz: {
+        maxHeight: 500
+    }
+};
+
+
+class Cardz extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {tilesData: []};
+};
+
+  handleToggle(toggle, index, event) {
+    var newState = update(this.state, {tilesData: {[index]: {expanded: {$set: toggle}}}})
```

Comment:
The `[index]` syntax is strange here and relies on an iffy bit of casting from an array to a string.  You should do this in two steps.